### ISSUE TITLE
NavigationBar Items are Not Showing after referencing

### ISSUE THREAD

**Issue by KathirveluM** [rotorgames/Rg.Plugins.Popup#18]
Hi @rotorgames ,

The popup page are showing/working properly after adding the Nuget Reference in iOS and Android Platforms.
Issue is, 
Actionbar items are shown properly in Android Platform but in iOS Platform , NavigationBar items are not showing after adding the Rg.Plugins.PopUp reference. 

This is for your information:
Version Visual Studio : 4.0.4.4
Version Xamarin studio: 5.10.3(build 51)
Version Xamarin Forms: 2.2.0.31
Xamarin iOS Version : 9.6.2.2
Popup plugin version:0.0.16
[Ref Image](https://onedrive.live.com/redir?resid=79CF7AC39730AFB9!1066&authkey=!AJUFBQJdZkcprEA&v=3&ithint=photo%2cpng)

**Comment by rotorgames** [comment]
@KathirveluM Hi friend. Please, send me you project, where you have problem.

**Comment by KathirveluM** [comment]
Hi, could you please provide me your emai ID..

**Comment by rotorgames** [comment]
@KathirveluM rotorgames93@gmail.com

**Comment by rotorgames** [comment]
@KathirveluM I'm open you project, and deleted Popup assamblyes in nuget. 
ToolbarItems still not showed. What is to blame my plugin?

Error in your code.

**Comment by KathirveluM** [comment]
Hi, 
Yes, you are right. But Before including the Rg.Popup Plugin Reference it was working as expected.

After including as well as deleting the reference , its not working in iOS platform.

**Comment by rotorgames** [comment]
@KathirveluM I tested on my Demo project. In him all working!

![clip2net_160524141210](https://cloud.githubusercontent.com/assets/1790504/15500406/a99f4158-21b9-11e6-932c-80096ab08106.png)

**Comment by KathirveluM** [comment]
Hi @rotorgames, 

I too have tested with sample application and its all working (Now too i have checked and its fine).

When we add the same with our working project its not showing the Navigationbar in iOS platform alone. 

I have verified the existing iOS projects (before and after referencing) and found that only difference is Package file is updated for Xamarin Forms Version from 2.0.1.6505 to 2.2.0.31 but its not an issue (because i have checked with fresh sample and its working fine with 2.2.0.31 version as well)

There is no issues with Component . i Feel after Package Updating only the issue arised. Kindly provide your valuable inputs on this.

**Comment by rotorgames** [comment]
@KathirveluM Rg.Plugins.Popup depends only from Xamarin.Forms. 

I not find error in your code, but code dirty.
Why use `try catch` everywhere, and without trapping exception? 

You can create new blank project, add my plugin, and repeat you code, by landmark.

**Comment by KathirveluM** [comment]
Hi @rotorgames , Ok let me start from the scratch with new project and check it. 

Where do you mean dirty (The try Catch blocks?) or any other ?

**Comment by rotorgames** [comment]
@KathirveluM Dirty in all `try` `catch` code.

You not catch exception. When throw error, you dont observe about it.

don't write this:
```csharp
try
{
//you code
}
catch(Exception)
{
// This nothing
// You dont observe when error throwing
}
```

**Comment by KathirveluM** [comment]
@rotorgames Yes, we have to throw the exceptions properly in try catch block and this is not yet done for this application. 

The reason for this issue :

As per our requirement, we have aligned the First Navigation bar item as left and other items as right for all the Xamarin Forms content pages using Custom Renderer (ExtendedPageRenderer.cs). 

Now We have unloaded this Class and its working. 

FYI.
The file can be found in AppnameiOS->Renderer -> ExtendedPageRenderer.cs
